src/components/UserAccess/UserAccessForm/form-utils.ts

```diff
@@ -30,29 +20,60 @@ export const userAccessFormSchema = yup.object({
     .required('Required.'),
 });
 
-export const createSBRs = async (
+export const getRBs = (namespace: string, dryRun?: boolean): Promise<RoleBinding[]> => {
+  return K8sListResourceItems({
+    model: RoleBindingModel,
+    queryOptions: {
+      ns: namespace,
+      ...(dryRun && { queryParams: { dryRun: 'All' } }),
+    },
+  })
+    .then((res) => {
+      const roleBindings: RoleBinding[] = res as RoleBinding[];
+      return roleBindings;
+    })
```

Comment:
This method is just performing typecasting on the returned response. Don't need this